snaps-sidebar.js

```diff
@@ -22,10 +22,7 @@ const sidebar = {
       label: "Get started",
       link: { type: "generated-index" },
       collapsed: false,
-      items: [
-        "get-started/install-snaps",
-        "get-started/quickstart",
-      ],
+      items: ["get-started/install-snaps", "get-started/quickstart"],
```

Comment:
Looks like the preferred formatting was broken here. Maybe there is a linting command to fix it? This change should be reverted.